skimage/feature/corner.py

```diff
@@ -921,6 +921,11 @@ def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=0.1,
     This differs from `skimage.feature.peak_local_max` in that it suppresses
     multiple connected peaks with the same accumulator value.
 
+    Note that the `num_peaks` limit is applied *before* suppression of
```

Comment:
I think this paragraph should rather go into `Notes` section.

Backticks should be doubled (see https://github.com/scikit-image/scikit-image/issues/4117), asterisks - removed.

Otherwise, 👍 .

Reply:
I'm +1 for the note section. But please, keep simple backticks. We fixed the issue and they are correctly handled now #4188